guides/common/modules/proc_configuring-dhcpd-to-use-with-server.adoc

```diff
@@ -0,0 +1,182 @@
+[id="configuring-dhcpd-to-use-with-server"]
+= Configuring dhcpd to use with {ProductName}
```

Comment:
Why is that "ProductName" in favor of "ProjectServer"?

Reply:
Agreed, can you please s/ProductName/ProjectServer/ everywhere?
```suggestion
= Configuring dhcpd to use with {ProjectServer}
```
See also https://github.com/theforeman/foreman-documentation/pull/3530#discussion_r2060301470

Reply:
Technically it should really be either `ProjectServer` or `SmartProxyServer` if you go the server route, but perhaps it can be more generic here and mention the project name instead?

Though I will again note that the remote ISC DHCP server really has many issues already and perhaps better to track it as a follow up.